FIX: Issue when deployed on Elastic BeanStalk without KEYMAT

dhilton opened the pull request:
**What**

When deploying survey runner without KEYMAT(public, private keys for JWT) the survey errors as it expects the KEYMAT to be set to file paths not none as os.get.environ returns when an a env var is not found. 

This PR adds an explicit test for the setting of those variables and raises a specific exception if there is not one set. It also removes an unneccesary import that was suppressing the root cause. 

**How to test**
1. Checkout this branch
2. Run `./scripts/run_app.sh` - this will automatically set the KEYMAT variables.
3. Run in the root dir `python application.py` and the script should error out raising an OSError.

**Who can review**

Anyone but @dhilton.

warren-methods commented:
Looks good, but @weapdiv-david has another branch ui-integration which has the same changes along with some functional changes. I think it's worth catching up with him before merging this.

dhilton committed:
FIX: When KEYMAT not configured, None is passed to CFFI causing errors

When the application is deployed to an ElasticBeanstalk environment and the SR_ENVIRONMENT variable is set,
the application looks for the presence of two env vars EQ_RRM_PUBLIC_KEY and EQ_SR_PRIVATE_KEY. If these
are set to None then the application fails, as it can't initialise a Decoder correctly for JWT usage.

This commit adds a check for the presence of those two settings, raising an OSError (closest error type ish) to
highlight the error as soon as it appears.

This error (and all errors) are currently surpressed by incorrect HTTP 500 error handlers that hampered finding this bug.
The next commit will address this issue.

dhilton committed:
remove unneeded import that was surpressing web errors.
